nixos/modules/misc/ids.nix

```diff
@@ -339,6 +339,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      corends = 315;
```

Comment:
typo: you must have meant `coredns`

It's the same for the group id

Reply:
Ah yeah thanks :smile: